src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -238,11 +245,11 @@ private void validateMedicalInventory(MedicalInventory medInventory) throws OHDa
 	 *
 	 * @param inventory - The {@link MedicalInventory}
 	 * @param inventoryRowSearchList- The list of {@link MedicalInventory}
-	 * @throws OHDataValidationException
+	 * @throws OHDataValidationException, OHServiceException
 	 */
 	public void validateMedicalInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList)
 					throws OHDataValidationException, OHServiceException {
```

Comment:
Remove `OHDataValidationException` from the `throws` list as it is subclass of `OHServiceException`